code/missionui/missionshipchoice.cpp

```diff
@@ -1929,6 +1930,11 @@ void commit_pressed()
 	if (!(The_mission.flags[Mission::Mission_Flags::No_briefing]))
 		gamesnd_play_iface(InterfaceSounds::COMMIT_PRESSED);
 
+	// plieblang - resume simulated speech if necessary
+	if (!Player->auto_advance) {
+		fsspeech_pause(Player->auto_advance);
```

Comment:
Since `Player->auto_advance` is already known to be `false` here could you pass `false` explicitly as the parameter?

```suggestion
		fsspeech_pause(false);
```